### ISSUE TITLE
page_navbar() tab controls are missing aria-controls attributes

### ISSUE THREAD

**Issue by shelbylevel** [rstudio/bslib#1354]
## Describe the problem

When using `bslib::page_navbar() with bslib::nav_panel()`, the generated tab controls with a `role="tab"` attribute do not include an `aria-controls` attribute to establish an explicit relationship between the tab and the controlled tab panel. This is causing accessibility scanners to flag the tabs as an incomplete implementation of the WAI-ARIA Tabs pattern. Need to ensure the value of the `aria-controls` attribute is the id of the tabpanel it is intended to control. 

<img width="1468" height="852" alt="Image" src="https://github.com/user-attachments/assets/c9c39cd0-c39e-4380-9b01-ef44cb3bfb86" />

## Reprex

```r
library(shiny)
library(bslib)

ui <- page_navbar(
  title = "Example",
  nav_panel("Welcome", "Welcome content"),
  nav_panel("Tab 2", "Tab 2 content")
)

server <- function(input, output, session) {}

shinyApp(ui, server)
```

Right click > Inspect on nav tab name

### Session Info

<details>
<pre><code>
─ Session info ────────────────────────────────────────────────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.5.1 (2025-06-13 ucrt)
 os       Windows Server 2022 x64 (build 20348)
 system   x86_64, mingw32
 ui       Positron
 language (EN)
 collate  English_United States.utf8
 ctype    English_United States.utf8
 tz       America/New_York
 date     2026-09-17
 pandoc   3.10.2 @ E:\\Pandoc\\pandoc.exe
 quarto   ERROR: Unknown command "TMPDIR=C:/Users/leve44/AppData/Local/Temp/51/RtmpIhFYLp/filee0f01af2134f". Did you mean command "preview"? @ E:\\Quarto\\bin\\quarto.exe

─ Packages ────────────────────────────────────────────────────────────────────────────────────────────────────────────
 package     * version     date (UTC) lib source
 brand.yml     0.1.0       2025-11-06 [1] CRAN (R 4.5.2)
 bslib       * 0.12.0.9000 2026-09-17 [1] Github (rstudio/bslib@7d9e2db)
 cachem        1.1.0       2024-05-16 [1] CRAN (R 4.5.1)
 callr         3.7.6       2024-03-25 [1] CRAN (R 4.5.1)
 cli           3.6.6       2026-04-09 [1] CRAN (R 4.5.3)
 curl          7.0.0       2025-08-19 [1] CRAN (R 4.5.1)
 desc          1.4.3       2023-12-10 [1] CRAN (R 4.5.2)
 devtools      2.4.6       2025-10-03 [1] CRAN (R 4.5.2)
 digest        0.6.39      2025-11-19 [1] CRAN (R 4.5.2)
 ellipsis      0.3.2       2021-04-29 [1] CRAN (R 4.5.2)
 fastmap       1.2.0       2024-05-15 [1] CRAN (R 4.5.1)
 fs            2.1.0       2026-04-18 [1] CRAN (R 4.5.3)
 glue          1.8.0       2024-09-30 [1] CRAN (R 4.5.1)
 htmltools     0.5.9       2025-12-04 [1] CRAN (R 4.5.2)
 httpuv        1.6.17      2026-03-18 [1] CRAN (R 4.5.3)
 jquerylib     0.1.4       2021-04-26 [1] CRAN (R 4.5.1)
 jsonlite      2.0.0       2025-03-27 [1] CRAN (R 4.5.1)
 later         1.4.8       2026-03-05 [1] CRAN (R 4.5.2)
 lifecycle     1.0.5       2026-01-08 [1] CRAN (R 4.5.2)
 magrittr      2.0.4       2025-09-12 [1] CRAN (R 4.5.2)
 memoise       2.0.1       2021-11-26 [1] CRAN (R 4.5.1)
 mime          0.13        2025-03-17 [1] CRAN (R 4.5.0)
 otel          0.2.0       2025-08-29 [1] CRAN (R 4.5.3)
 pillar        1.11.1      2025-09-17 [1] CRAN (R 4.5.2)
 pkgbuild      1.4.8       2025-05-26 [1] CRAN (R 4.5.2)
 pkgconfig     2.0.3       2019-09-22 [1] CRAN (R 4.5.1)
 pkgload       1.5.0       2026-02-03 [1] CRAN (R 4.5.2)
 processx      3.8.6       2025-02-21 [1] CRAN (R 4.5.1)
 promises      1.5.0       2025-11-01 [1] CRAN (R 4.5.3)
 ps            1.9.1       2025-04-12 [1] CRAN (R 4.5.1)
 purrr         1.2.1       2026-01-09 [1] CRAN (R 4.5.2)
 R6            2.6.1       2025-02-15 [1] CRAN (R 4.5.1)
 Rcpp          1.1.1-1.1   2026-04-24 [1] CRAN (R 4.5.3)
 remotes       2.5.0       2024-03-17 [1] CRAN (R 4.5.1)
 rlang         1.3.0       2026-07-05 [1] CRAN (R 4.5.3)
 sass          0.4.10      2025-04-11 [1] CRAN (R 4.5.1)
 sessioninfo   1.2.3       2025-02-05 [1] CRAN (R 4.5.2)
 shiny       * 1.14.0      2026-06-21 [1] CRAN (R 4.5.3)
 tibble        3.3.1       2026-01-11 [1] CRAN (R 4.5.2)
 usethis       3.2.1       2025-09-06 [1] CRAN (R 4.5.3)
 vctrs         0.7.1.9000  2026-03-17 [1] Github (r-lib/vctrs@84f645a)
 xtable        1.8-8       2026-02-22 [1] CRAN (R 4.5.3)

 [1] C:/Users/leve44/AppData/Local/R/win-library/4.5
 [2] E:/R/R-4.5.1/library
 * ── Packages attached to the search path.
</code></pre>
</details>
